includes/html/pages/device-dependencies.inc.php

```diff
@@ -81,6 +81,9 @@
         "hostname": function(column, row) {
             return '<a href="device/device='+row.deviceid+'/" class="list-device">'+row.hostname+'</a><br />'+row.sysname;
         },
+        "deviceid": function(column, row) {
```

Comment:
this is not needed if youre not formatting the data

Reply:
Ok, thanks.